test/lib/edemam_wrapper.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
```

Comment:
This filename should be `edamam_wrapper_test.rb`, it won't run the tests automatically.